### PR TITLE
Add hidden elements

### PR DESCRIPTION
**Pull request by rex-iotum**
Add the ability to pass in an array of numbers to hide UI elements

Didn't physically test in dev but added a test case and checked the integration on Facetalk's side.
image

### CODE REVIEW

File: src/dashboard.ts [hunk 1]
```diff
@@ -61,6 +61,10 @@ export type ServiceOptions = {
    * The meeting action.
    */
   meetingAction?: MeetingAction;
+  /**
+   * the UI elements to be hidden.
+   */
+  hiddenElements?: number[];
```
Comment (jerry2013) [inline src/dashboard.ts:67]: discussion, as an API - do we want an enum for the possible values?

**Comment by jerry2013** [comment]
Also, you'll need to update package version (package.json) + build then commit the /dist (npm build)

File: src/dashboard.ts (hunk 1 above)
Comment (rex-iotum) [reply src/dashboard.ts:67]: We can direct them to enable the UI Map on facetalk via feature flag dev tools to see which elements gets hidden. Imo enum would just be listing 50+ internal flag names that doesn't fully describe or show what will be hidden.
image

**Comment by rex-iotum** [comment]
Done, updated version and built the /dist

File: src/dashboard.ts (hunk 1 above)
Comment (jerry2013) [reply src/dashboard.ts:67]: are you saying the same 'hidden elements' param is also intended for the meeting (call) widget?

Comment (rex-iotum) [reply src/dashboard.ts:67]: No, atm its only for the dashboard. Apparently the InCallPage has a different provider for the hidden elements.

**Review by jerry2013 — APPROVED** [review]

**Pull request merged**